pkg/operator/staticpod/controller/backingresource/backing_resource_controller.go

```diff
@@ -144,7 +144,7 @@ func (c BackingResourceController) sync() error {
 			glog.Error(updateError)
 		}
 	}
-	return err
+	return fmt.Errorf("synthetic requeue (errs: %q)", strings.Join(errors, ","))
```

Comment:
@deads2k unit tests are useful :-) this will report nil and cause the controller not to resync when the resource creation fail.